lib-xml/src/main/java/com/orange/ods/xml/component/banner/OdsBanner.kt

```diff
@@ -21,40 +21,45 @@ import androidx.compose.runtime.setValue
 import androidx.core.content.withStyledAttributes
 import com.google.accompanist.drawablepainter.rememberDrawablePainter
 import com.orange.ods.compose.component.banner.OdsBanner
+import com.orange.ods.compose.component.banner.OdsBannerButton
+import com.orange.ods.compose.component.banner.OdsBannerImage
+import com.orange.ods.utilities.extension.ifNotNull
 import com.orange.ods.xml.R
 import com.orange.ods.xml.component.OdsAbstractComposeView
 import com.orange.ods.xml.utilities.extension.getResourceIdOrNull
 
 class OdsBanner @JvmOverloads constructor(context: Context, attrs: AttributeSet? = null) : OdsAbstractComposeView(context, attrs) {
 
     var message by mutableStateOf("")
-    var button1Text by mutableStateOf("")
-    var onButton1Click by mutableStateOf({})
+    var firstButtonText by mutableStateOf("")
+    var onFirstButtonClick by mutableStateOf({})
     var image by mutableStateOf<Drawable?>(null)
     var imageContentDescription by mutableStateOf<String?>(null)
-    var button2Text by mutableStateOf<String?>(null)
-    var onButton2Click by mutableStateOf<(() -> Unit)?>(null)
+    var secondButtonText by mutableStateOf<String?>(null)
+    var onSecondButtonClick by mutableStateOf<(() -> Unit)?>(null)
 
     init {
         context.withStyledAttributes(attrs, R.styleable.OdsBanner) {
             message = getString(R.styleable.OdsBanner_message).orEmpty()
-            button1Text = getString(R.styleable.OdsBanner_button1Text).orEmpty()
+            firstButtonText = getString(R.styleable.OdsBanner_firstButtonText).orEmpty()
             image = getResourceIdOrNull(R.styleable.OdsBanner_image)?.let { AppCompatResources.getDrawable(context, it) }
             imageContentDescription = getString(R.styleable.OdsBanner_imageContentDescription)
-            button2Text = getString(R.styleable.OdsBanner_button2Text)
+            secondButtonText = getString(R.styleable.OdsBanner_secondButtonText)
         }
     }
 
     @Composable
     override fun OdsContent() {
         OdsBanner(
             message = message,
-            button1Text = button1Text,
-            onButton1Click = onButton1Click,
-            image = image?.let { rememberDrawablePainter(drawable = it) },
-            imageContentDescription = imageContentDescription,
-            button2Text = button2Text,
-            onButton2Click = onButton2Click
+            firstButton = OdsBannerButton(firstButtonText, onFirstButtonClick),
+            image = ifNotNull(image, imageContentDescription) { image, contentDescription ->
```

Comment:
In ODS demo app, the image isn't displayed in the XML banner cause the `imageContentDescription` is `null`. 
I don't know if we have to display the image in this case but the user doesn't know why his image is not displayed.